Jieun/app.js

```diff
@@ -54,9 +54,8 @@ app.post('/users/sign-up', async (req, res, next) => {
 
 
 app.post('/posts/register', async (req, res, next) => {
```

Comment:
```suggestion
app.post('/posts', async (req, res, next) => {
```
엔드포인트 끝에, `/register`라는 추가적인 작성이 이루어진것은 등록을 의미하는 것 같습니다. 엔드포인트에는 어떠한 액션(동작)이 들어가지 않도록 작성되는게 REST한 api end-point를 설계하는것입니다. 액션(동작)은 http request method를 통해, 가져오고 등록하고 수정하고 삭제하는 것을 충분히 나타낼 수 있기 때문입니다. 자세한 내용은 REST 세션에서 다루겠지만 지금은 엔드포인트에 동사가 들어가지 않도록 해야한다는 것만 알아주세요!

해당 엔드포인트 외에 다른 api의 엔드포인트들도 모두 비슷한 맥락의 리뷰가 적용될 수 있는데, 모두 수정 부탁드리겠습니다 :)